src/pipecat/services/asyncai/tts.py

```diff
@@ -0,0 +1,513 @@
+            sample_rate: Audio sample rate.
+            encoding: Audio encoding format.
+            container: Audio container format.
+            params: Additional input parameters for voice customization.
+            aggregate_sentences: Whether to aggregate sentences within the TTSService.
+            **kwargs: Additional arguments passed to the parent service.
+        """
+        # Aggregating sentences still gives cleaner-sounding results and fewer
+        # artifacts than streaming one word at a time. On average, waiting for a
+        # full sentence should only "cost" us 15ms or so with GPT-4o or a Llama
+        # 3 model, and it's worth it for the better audio quality.
+        #
+        # We also don't want to automatically push LLM response text frames,
+        # because the context aggregators will add them to the LLM context even
+        # if we're interrupted.
+        super().__init__(
+            aggregate_sentences=aggregate_sentences,
+            push_text_frames=False,
+            pause_frame_processing=True,
+            sample_rate=sample_rate,
+            **kwargs,
+        )
+
+        params = params or AsyncAITTSService.InputParams()
+
+        self._api_key = api_key
+        self._api_version = version
+        self._url = url
+        self._settings = {
+            "output_format": {
+                "container": container,
+                "encoding": encoding,
+                "sample_rate": sample_rate,
+            },
+            "language": self.language_to_service_language(params.language)
+            if params.language
+            else "en",
+        },
+        
+        self.set_model_name(model)
+        self.set_voice(voice_id)
+        self._global_context_id = str(uuid.uuid4())
+
+        self._context_id = None
+        self._receive_task = None
+        self._keepalive_task = None
+
+    def can_generate_metrics(self) -> bool:
+        """Check if this service can generate processing metrics.
+
+        Returns:
+            True, as Async service supports metrics generation.
+        """
+        return True
+
+    def language_to_service_language(self, language: Language) -> Optional[str]:
+        """Convert a Language enum to Async language format.
+
+        Args:
+            language: The language to convert.
+
+        Returns:
+            The Async-specific language code, or None if not supported.
+        """
+        return language_to_async_language(language)
+
+    def _build_msg(
+        self, text: str = "", force: bool = False
+    ) -> str:
+        msg = {
+            "transcript": text,
+            "force": force
+        }
+        return json.dumps(msg)
+    
+    async def start(self, frame: StartFrame):
+        """Start the Async TTS service.
+
+        Args:
+            frame: The start frame containing initialization parameters.
+        """
+        await super().start(frame)
+        await self._connect()
```

Comment:
Initialize sample rate. This allows it to get the StartFrame value.
```suggestion
        self._settings["output_format"]["sample_rate"] = self.sample_rate
        await self._connect()
```